src/cpx_constrs.jl

```diff
@@ -165,12 +192,31 @@ function set_constr_senses!(model::Model, senses::Vector)
                       Ptr{Cint},
                       Ptr{Cchar}
                       ),
-                      model.env.ptr, model.lp, ncons, Cint[0:ncons-1;], convert(Vector{Cchar},senses))
+                      model.env.ptr, model.lp, ncons, Cint[0:ncons-1;], 
+                      convert(Vector{Cchar},senses))
     if stat != 0
         throw(CplexError(model.env, stat))
     end
 end
 
+function c_api_getrhs(model::Model, row_start::Cint, row_end::Cint)
```

Comment:
I think users should have to pass in the `rhs` vector as well.
```julia
function c_api_getrhs(model::Model, rhs::Vector{Cdouble}, row_start::Cint, row_end::Cint)
```

Reply:
@odow, I agree with you. higher layers can benefit from that doing in place modification even if it's not the case of LQOI for this specific function. I will update it.